resources/javascripts/login.js

```diff
@@ -0,0 +1,56 @@
+var validation = false;

```

Comment:
having this as a global variable can lead to intentional/unintentional modification of this variable and thus leading to incorrect behaviour.